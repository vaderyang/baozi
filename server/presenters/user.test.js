/* eslint-disable flowtype/require-valid-file-annotation */
import presentUser from "./user";

it("presents a user", async () => {
  const user = presentUser({
    id: "123",
    name: "Test User",
    username: "testuser",
  });

  expect(user).toMatchSnapshot();
});

it("presents a user without slack data", async () => {
  const user = presentUser({
    id: "123",
    name: "Test User",
    username: "testuser",
  });

  expect(user).toMatchSnapshot();
});
