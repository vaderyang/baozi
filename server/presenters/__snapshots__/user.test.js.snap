// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`presents a user 1`] = `
Object {
  "avatarUrl": undefined,
  "createdAt": undefined,
  "id": "123",
  "isAdmin": undefined,
  "isSuspended": undefined,
  "language": "en_US",
  "lastActiveAt": undefined,
  "name": "Test User",
}
`;

exports[`presents a user without slack data 1`] = `
Object {
  "avatarUrl": undefined,
  "createdAt": undefined,
  "id": "123",
  "isAdmin": undefined,
  "isSuspended": undefined,
  "language": "en_US",
  "lastActiveAt": undefined,
  "name": "Test User",
}
`;
